lib/webpacker/package_json.rb

```diff
@@ -0,0 +1,58 @@
+# Loads the package.json file from app root to read the webpacker configuration
+
+class Webpacker::PackageJson
```

Comment:
I'd go with `Webpacker::Configuration` instead. The fact that the config is in package.json is an implementational detail.